src/storage/lsm.rs

```diff
@@ -44,7 +44,6 @@ where
         ));
         let commit_log = OpenOptions::new()
             .create(true)
-            .write(true)
```

Comment:
We don't need to specify write and append both.  If we had write only, then file would be overwritten from the beginning, corrupting it.